screener-e2e/examples/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "examples",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test.screener.local": "npx wdio test/configs/wdio.screener.local.conf.js",
+    "test.screener.sauce": "npx wdio test/configs/wdio.screener.saucelabs.conf.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@wdio/cli": "^6.3.0",
+    "@wdio/jasmine-framework": "^6.0.16",
+    "@wdio/local-runner": "^6.3.0",
+    "@wdio/mocha-framework": "^6.3.0",
+    "@wdio/spec-reporter": "^6.3.0",
+    "@wdio/sync": "^6.3.0",
+    "chromedriver": "^83.0.1",
+    "wdio-chromedriver-service": "^6.0.3"
```

Comment:
You didn't add the `@wdio/sauce-service` in the dependencies and config, see https://webdriver.io/docs/sauce-service.html for more info

Reply:
Hey bud, I don't think I need it, everything seems to work without it

Reply:
Ok, it seems to work, but that has to do with the `hostname: 'staging-hub.screener.io',` in the shared config. You will get this error in the logs because the service is not there

```log

2020-07-17T15:55:01.156Z DEBUG @wdio/utils:initialiseServices: initialise service "sauce" as NPM package
2020-07-17T15:55:01.158Z ERROR @wdio/utils:initialiseServices: Error: Couldn't find plugin "sauce" service, neither as wdio scoped package "@wdio/sauce-service" nor as community package "wdio-sauce-service". Please make sure you have it installed!
    at initialisePlugin (/Users/wimselles/Sauce/Git/sauce-training/demo-js/screener-e2e/examples/node_modules/@wdio/utils/build/initialisePlugin.js:32:9)
    at initialiseServices (/Users/wimselles/Sauce/Git/sauce-training/demo-js/screener-e2e/examples/node_modules/@wdio/utils/build/initialiseServices.js:34:51)
    at initialiseLauncherService (/Users/wimselles/Sauce/Git/sauce-training/demo-js/screener-e2e/examples/node_modules/@wdio/utils/build/initialiseServices.js:50:22)
    at Launcher.run (/Users/wimselles/Sauce/Git/sauce-training/demo-js/screener-e2e/examples/node_modules/@wdio/cli/build/launcher.js:73:48)
    at launch (/Users/wimselles/Sauce/Git/sauce-training/demo-js/screener-e2e/examples/node_modules/@wdio/cli/build/commands/run.js:138:19)
    at handler (/Users/wimselles/Sauce/Git/sauce-training/demo-js/screener-e2e/examples/node_modules/@wdio/cli/build/commands/run.js:171:12)
    at Object.run (/Users/wimselles/Sauce/Git/sauce-training/demo-js/screener-e2e/examples/node_modules/@wdio/cli/build/index.js:47:29)
    at Object.<anonymous> (/Users/wimselles/Sauce/Git/sauce-training/demo-js/screener-e2e/examples/node_modules/@wdio/cli/bin/wdio.js:11:21)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
```

I've pushed the service and will merge this PR